src/Batch/SetupBatchIntegrationTestEnvironment.ps1

```diff
@@ -0,0 +1,93 @@
+﻿Param([switch] $Preview) 
```

Comment:
Preview might be better called "Show" or "Display"?

I saw preview and thought it was enabling some preview features for a second...

With that said, that may have just been a brain fart so feel free to ignore this